internals/webhooks/webhooks_messages.go

```diff
@@ -0,0 +1,6 @@
+package webhooks
+
+const (
+	denyMessage  = "Delete non-leaf namespace is forbidden"
```

Comment:
deny message should be more clear to the end-user, maybe something like this:
`Deleting a Namespace with Subnamesapces is forbidden.`

Reply:
That's right, agree.